shell/platform/windows/client_wrapper/flutter_engine_unittests.cc

```diff
@@ -74,6 +83,8 @@ class TestFlutterWindowsApi : public testing::StubFlutterWindowsApi {
     next_frame_callback_ = nullptr;
   }
 
+  UINT last_message() { return last_message_; }
```

Comment:
Consider calling this `last_external_message`. I don't feel strongly, feel free to skip

```suggestion
  UINT last_external_message() { return last_external_message_; }
```